sfm_pc/management/commands/import_google_doc.py

```diff
@@ -121,22 +113,19 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument(
-            '--doc_id',
-            dest='doc_id',
-            default='1o-K13Od1pGc7FOQ-JS8LJW9Angk3_UPZHTmOU7hD3wU',
-            help='Import data from specified Google Drive Document'
+            '--country_code',
+            dest='country_code',
+            help='Country code for the import'
         )
         parser.add_argument(
-            '--source_doc_id',
-            dest='source_doc_id',
-            default='15lChqP4cKsjk8uUbUTaUA5gOUM09-t9YMIODORHw--8',
-            help='Import source data from specified Google Drive Document'
+            '--country_directory',
```

Comment:
Can you standardize on directory or path for the country data and source file arg? Probably path, since it works for both directories and files.